src/main/java/org/killbill/billing/entitlement/api/Entitlement.java

```diff
@@ -340,7 +340,7 @@ public Entitlement changePlanWithDate(final EntitlementSpecifier spec, final Loc
      * @throws EntitlementApiException if change failed
      */
     @RequiresPermissions(ENTITLEMENT_CAN_CHANGE_PLAN)
-    public Entitlement changePlanWithDate(final EntitlementSpecifier spec, final DateTime effectiveDate, final Iterable<PluginProperty> properties, final CallContext context)
+    public Entitlement changePlanWithDateTime(final EntitlementSpecifier spec, final DateTime effectiveDate, final Iterable<PluginProperty> properties, final CallContext context)
```

Comment:
Naming is hard... if we introduce `xxxWithdateTime`, then should we follow the same naming pattern for `cancelEntitlementWithDate` ?